databroker/tests/test_pipelines.py

```diff
@@ -0,0 +1,112 @@
+import time
+import tempfile
+import os
+import uuid
+from functools import partial
+import numpy as np
+from databroker.broker import store_dec, event_map, header_io
+
+
+class NpyWriter:
```

Comment:
I think it might be nice to eventually have default writers in databroker, just as you do handlers. `NPYwriter`looks fine to me as it is now. Also, this used to be an 'up to the user' part of data acquisition (see README.md [here](https://github.com/NSLS-II/fileStore)).
`store_dec` is now breaking that assumption. I think it's nice in that it's completing a missing void. If you add these writers, I think you officially declare support for the storage (which as a user I like :) ). Just a thought....